### DISCUSSION TITLE
Draft Pre-Release is empty

### DISCUSSION THREAD

**Discussion by Falke-Design** [community/community#33740] [Mobile]
When I open a Draft Pre-Release in the App, nothing is shown. Is this a wanted "feature" or a bug?

**Comment by eliperkins** [comment]
Hey @Falke-Design , thanks for the feedback. That sounds like a bug to me, as drafts and pre-releases both should show content within the. Would you mind giving me a little bit more information to help debug this further?

- Do you have a screenshot of the "nothing" state? It might help us understand the bug better to see the empty state
- How are you navigating to the release? From a deep link or from Repo Profile > Releases > Release?
- What version of the app, OS, and what device are you on? This will help us reproduce the bug as well.

Thanks!

**Reply by Falke-Design** [reply]
I need to correct me, only drafts are a problem. Pre-releases are working.

I recorded my screen:

https://user-images.githubusercontent.com/19800037/191894097-72f8edc4-c899-42ac-bb02-9d0748c6a3b4.mp4

Here the screen on the desktop:
![grafik](https://user-images.githubusercontent.com/19800037/191894521-ff29fcd1-89f0-4017-85b6-26d39fd56155.png)


> How are you navigating to the release? From a deep link or from Repo Profile > Releases > Release?

Yes I navigate from the repo to Releases and then I click on the Release.


> What version of the app, OS, and what device are you on? This will help us reproduce the bug as well.

GitHub Mobile v1.79.0 (675)
Android 12
Samsung Galaxy S20 FE

**Reply by Emad3010** [reply]
Emad
